### PR TITLE
bump versions of the toolchain dependencies

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

**Review by adrianchiris — APPROVED** [review]
LGTM, just need to make CI happy

**Comment by tariq1890** [comment]
Thanks @adrianchiris .  Hopefully, the CI should be happy now. Requesting another review

File: .golangci.yaml [hunk 1]
```diff
@@ -92,6 +92,16 @@ linters:
...
+  depguard:
+    rules:
+      main:
+        list-mode: original
+        allow:
+          - $gostd
+          - github.com/NVIDIA
+          - github.com/go-logr/logr
+          - k8s.io
+          - sigs.k8s.io
```
Comment (cdesiniotis) [inline .golangci.yaml:104]: Question -- why was this change made as part of this PR?

Comment (tariq1890) [reply .golangci.yaml:104]: `depguard` was one of the linters enabled on this repo and it started failing with the latest golangci-lint versions. Adding these depguard settings allow lists all of the dependencies imported by k8s-operator-libs

**Review by rollandf — APPROVED** [review]

**Review by cdesiniotis — APPROVED** [review]

**Pull request merged**